test/PSRP.Tests.ps1

```diff
@@ -0,0 +1,138 @@
+Describe " PowerShell Remoting basic functional tests" -Tag @("CI") {
```

Comment:
This test file has to be run on three different platforms to cover all the tests. Consider mentioning that in the Readme.